components/toshiba/TLP3409S.stanza

```diff
@@ -0,0 +1,127 @@
+#use-added-syntax(jitx)
+defpackage ocdb/toshiba/TLP3409S:
+  import core
+  import collections
+  import math
+  import jitx
+  import jitx/commands
+  import ocdb/defaults
+
+  import ocdb/land-patterns
+  import ocdb/symbols
+  import ocdb/box-symbol
+  import ocdb/bundles
+  import ocdb/generator-utils
+  import ocdb/generic-components
+
+public pcb-component component :
+  manufacturer = "ON Semiconductor"
+  description = "SSR RELAY SPST-NO 650MA 0-100V"
+  mpn = "TLP3409S(TPL,E"
+
+  pin a 
+  pin c 
+  port drain : pin[[1 2]]
+
+  val sym = TLP3409S-symbol
+  val land = tlp-3409
+  symbol = sym(a => sym.a, c => sym.c, drain[1] => sym.drain[1], drain[2] => sym.drain[2])
+  landpattern = land(a => land.p[1], c => land.p[2], drain[1] => land.p[4], drain[2] => land.p[3])
+  property(self.forward-voltage) = 1.27
+  property(self.rds-on) = 0.4
+
+pcb-landpattern tlp-3409 :
+  for (l in grid-locs(2, 2, 0.8, 1.4), i in [2 1 3 4]) do :
+    pad p[i] : smd-pad(0.55, 1.0) at l
+
+  layer(Courtyard(Top)) = Rectangle(1.45, 2.0) 
+  layer(Silkscreen("pol", Top)) = Circle(1.0, 1.0, 0.254)
+
+  ref-label()
+
+
+pcb-symbol TLP3409S-symbol :
+  pin a at Point(-7.62, 2.54) with :
+    direction = Left
+    length = 2.54
+    number-size = 0.609
+    name-size = 0.609
+  pin c at Point(-7.62, -2.54) with :
+    direction = Left
+    length = 2.54
+    number-size = 0.609
+    name-size = 0.609
+  pin drain[1] at Point(7.62, 7.62) with :
+    direction = Right
+    length = 2.54
+    number-size = 0.609
+    name-size = 0.609
+  pin drain[2] at Point(7.62, -7.62) with :
+    direction = Right
+    length = 2.54
+    number-size = 0.609
+    name-size = 0.609
+  layer("foreground") = Text(">VALUE", 0.762, SW, loc(-7.62, 7.62 + 0.3))
+  layer("foreground") = Text(">REF", 0.762, SW, loc(-7.62, 7.62 + 0.3 + 0.762 + 0.3))
+  layer("foreground") = Circle(Point(5.08, 1.27), 0.355)
+  layer("foreground") = Circle(Point(5.08, 6.35), 0.355)
+  layer("foreground") = Polygon([Point(6.35, 4.318), Point(5.842, 3.556), Point(6.858, 3.556)])
```

Comment:
Would we like to have a `Triangle` shape?

Reply:
We have a unit-triangle function in symbol-utils, which could have been used for this.